tool_path_planner/src/plane_slicer_raster_generator.cpp

```diff
@@ -492,10 +524,11 @@ boost::optional<ToolPaths> PlaneSlicerRasterGenerator::generate()
   half_ext = sizes / 2.0;
   center = Eigen::Vector3d(bounds[0], bounds[2], bounds[3]) + half_ext;
 
-  // now cutting the mesh with planes along the y axis
-  // @todo This should be calculated instead of being fixed along the y-axis
+  // Apply the rotation offset about the short direction (new Z axis) of the bounding box
   Isometry3d rotation_offset = Isometry3d::Identity() * AngleAxisd(config_.raster_rot_offset, Vector3d::UnitZ());
-  Vector3d raster_dir = (rotation_offset * Vector3d::UnitY()).normalized();
+
+  // Calculate direction of raster strokes, rotated by the above-specified amount
+  Vector3d raster_dir = (rotation_offset * config_.raster_direction).normalized();
```

Comment:
I'm not sure what kind of results you would get if your raster_direction vector has more than one non-zero component or if it is (o, o, 1)?  Perhaps we should impose some restrictions on what values the raster_dir vector can take